docs/11-circleci/01-getting-started.mdx

```diff
@@ -0,0 +1,150 @@
+so.
+
+#### Unity Orb
+
+[Orbs](https://circleci.com/orbs/) are reusable snippets of code that help automate repeated
+processes, speed up project setup and make it easy to integrate with third-party tools.
+
+To streamline the process of testing and building Unity projects in CircleCI, GameCI provides the
+[Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity) — a certified orb capable of
+dependencies caching, artifacts storage, test data collection and macOS, Windows, Docker, Machine
+and Runner support.
+
+In further sections, you will learn how to import and use the orb to test and build your project for
+several target platforms.
+
+_**Note:** the orb is in the early stages of development; it doesn't have all features included in
+the [Builder Action](/docs/github/builder)._
+
+## Activation
+
+The orb uses a Unity Editor instance to test and build your projects. These operations demand the
+instance to be activated beforehand — to accomplish this, the orb requires an encoded license or
+serial to be included in the job’s Context. If you are unsure what this means or need help creating
+the license, navigate to the [Activation](/docs/circleci/activation) step before proceeding.
+
+> If you proceed to [set up your workflow](#setting-up-a-workflow) without going through
+> [activation](/docs/circleci/activation), your job won't have the necessary environment variables
+> for a successful run and it will fail.
+
+## Setting up a workflow
+
+Create a `.circleci/config.yml` file in the root of your repository and follow these steps:
+
+1. Import the [Unity Orb](https://circleci.com/developer/orbs/orb/game-ci/unity). It's recommended
+   to fix a `minor` or `patch` version to avoid updates with breaking changes:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+```
+
+2. Include a [build job](/docs/circleci/build) in your workflow and modify the
+   [parameters](/docs/circleci/build#parameters) to accommodate your project needs:
+
+```yaml
+version: 2.1
+
+orbs:
+  unity: game-ci/unity@x.y
+
+workflows:
+  build-unity-project:
+    jobs:
+      - unity/build:
+          name: 'build-linux64-il2cpp'
+          step-name: 'Build StandaloneLinux64'
+          unity-license-var-name: 'UNITY_ENCODED_LICENSE'
+          unity-username-var-name: 'UNITY_USERNAME'
+          unity-password-var-name: 'UNITY_PASSWORD'
+          executor:
+            name: 'unity/ubuntu'
+            target_platform: 'linux-il2cpp'
+            editor_version: '2021.3.1f1'
+            resource_class: 'large'
+          project-path: 'Unity2D-Demo-Game-CI-CD/src'
+          build-target: StandaloneLinux64
+          compress: true
+          context: unity
+```
+
+3. Optionally, include a [test job](https://circleci.com/developer/orbs/orb/game-ci/unity#jobs-test)
+   in your workflow. You can add it as a
+   [requirement](https://circleci.com/docs/configuration-reference#requires) in the build job to
+   prevent it from running in case of failure:
```

Comment:
The code block below this duplicates the entire build code-block.

Perhaps we could only add the part for the test, and a separate block with what has to change to the build step?

Reply:
Done ✔️